packages/viewer/src/components/scene-tree/scene-tree.css

```diff
@@ -139,6 +139,7 @@
   box-sizing: border-box;
   justify-content: center;
   background-color: white;
+  z-index: 3000;
```

Comment:
Curious here, any particular reason for `3000`? Also - one thought would be that it'd be nice to be able to override this in the future via CSS variables. I suspect this may not behave exactly as we expect in every application, so being able to adjust that easily would be nice

Reply:
Fair enough! It looks like I originally added those, so I'm guessing I pulled them from somewhere, but I can't recall exactly where... 🤔